incubator/fluentd-cloudwatch/templates/serviceaccount.yaml

```diff
@@ -9,3 +9,6 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 {{- end }}
+{{- if .Values.rbac.serviceAccountAnnotations }}
+  annotations: {{ toYaml .Values.rbac.serviceAccountAnnotations | nindent 4 }}
```

Comment:
shouldn't this be on a newline?

Reply:
I thought it weird too. But I used the same approach as with cluster autoscaler. Seems to work fine there:
https://github.com/helm/charts/blob/master/stable/cluster-autoscaler/templates/serviceaccount.yaml#L10

Reply:
To be sure I put it on a newline